src/types/callback.interface.ts

```diff
@@ -1 +1,8 @@
-export type EventCallback = (payload?: any) => void;
\ No newline at end of file
+import { IVerificationActionObj } from '~/types/verification-action.interface';
+
+export type EventCallback = (payload?: any) => void;
```

Comment:
Actually we can use Generic here 
```suggestion
export type EventCallback<T> = (payload?: T) => void;
```